### PR TITLE
MIGENG-189 WSR Scans Run

### PR DESCRIPTION
**Pull request by m-brophy**
To enable WSR ScanRunModel to contain scan type field

https://jira.coreos.com/browse/MIGENG-189

### CODE REVIEW

File: src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/Workloads.drl [hunk 1]
```diff
@@ -152,3 +152,35 @@ rule "Workloads_Microsoft_SQL_Server_On_Windows"
                addWorkload("Microsoft SQL Server")
            }
 end
+
+rule "Workloads_System_Services_Present"
```
Comment (mrizzi) [inline src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/Workloads.drl]: I know the rules and in the `Workload.drl` but we could use a slightly different name for them in order to help understanding they are not adding any workload.
```suggestion
rule "SsaEnabled_System_Services_Present"
```

File: src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/Workloads.drl [hunk 2]
```diff
@@ -152,3 +152,35 @@ rule "Workloads_Microsoft_SQL_Server_On_Windows"
...
+    then
+        modify(workloadInventoryReport)
+        {
+
+            setSsaEnabled(true)
+        }
+
+end
+
+rule "Workloads_System_Services_Not_Present"
```
Comment (mrizzi) [inline src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/Workloads.drl]: Like above
```suggestion
rule "SsaDisabled_System_Services_Not_Present"
```

**Review by mrizzi — COMMENTED** [review]
I figured out in the `BasicFields.drl` we don't set the `creationDate` field yet but it's mandatory for https://github.com/project-xavier/xavier-integration/pull/53 to work.
So please add the `workloadInventoryReport.setCreationDate(new Date());` (together with the `import java.util.Date`) in order to have the date available in the DB.

**Comment by m-brophy** [comment]
All items in review now addressed @mrizzi

**Review by mrizzi — APPROVED** [review]
This looks good and the test coverage is nice as well.
Thanks @m-brophy

**Pull request merged**